Pcap++/header/PcapFilter.h

```diff
@@ -1,5 +1,5 @@
-#ifndef PCAPP_FILTER
-#define PCAPP_FILTER
+#ifndef PCAP_FILTER
+#define PCAP_FILTER

```

Comment:
This should actually be `PCAPPP_FILTER` to meet the convention of the other file in Pcap++, but I suggest we leave it to a different PR. We should anyway replace these with `#pragma once` at some point...